nautobot/core/celery/task.py

```diff
@@ -8,10 +8,20 @@
 class NautobotTask(Task):
     """Nautobot extensions to tasks for integrating with Job machinery."""
 
-    def before_start(self, task_id, args, kwargs):
-        logger.debug(">>> TASK_ID = %s", task_id)
-        logger.debug(">>>    ARGS = %s", args)
-        logger.debug(">>>  KWARGS = %s", kwargs)
-
 
 Task = NautobotTask  # noqa: So that the class path resolves.
+
+
+# TODO(jathan): Remove this once this body of work is done. This is just useful for debugging but it
+# results int a lot of noise and slows things down.
+# from celery import signals
+# @signals.task_prerun.connect
```

Comment:
I just uncomment it locally when I need it for now. Leaving it in for testing generates a stupendous amount of noise.

I think documenting how to use it would be valuable.